site/tutorials/figures.md

```diff
@@ -0,0 +1,56 @@
+
+<img src="{{ site.baseurl }}/site/static/falcon.png" style="max-width: 500px">
+
+If you need to customize the figure beyond what is supported in Vega-Lite, you can open the compiled Vega in the editor, edit it as desired, and then continue the tutorial below.
+
+## Export as SVG
+
+Now that you have loaded the chart in the editor, you can export it with "Export" and then "Open SVG". We are exporting the chart as an SVG, a vector graphic format, which is infinitely scalable. You figure will be crisp even when zoomed in.
+
+<video controls style="max-width: 700px">
+  <source src="{{ site.baseurl }}/site/static/share.mp4" type="video/mp4">
+</video>
+
+## Print to PDF
+
+Unfortunately, Latex cannot import SVGs so we will have to convert it to PDF. There are many ways to do this: using illustrator, other image editor, command line scripts, or printing as PDF in the browser. You are free to use any of these tools if they fit your workflow but for this tutorial we are going to use the printing feature of your browser. To do this, select "File" and then "Print..." (or use <kbd>Cmd</kbd>+<kbd>P</kbd>) in your Chrome or Firefox browser.
+
+Now make sure that the destination is set to "Save as PDF" (see below). Then save the PDF in the directory of your Latex paper.
+
+<img src="{{ site.baseurl }}/site/static/print.png" style="max-width: 700px">
+
+## Crop the PDF
+
+You will notice that there is a lot of white space that we want to remove. To do this, open the PDF file in Mac OS Preview application. The select the "Rectangular Selection" tool and draw a box around the chart. You can adjust the box until it tightly fits the chart. Then click "crop" (or use <kbd>Cmd</kbd>+<kbd>K</kbd>). Make sure to save the file!
```

Comment:
```suggestion
You will notice that there is a lot of white space that we want to remove because the SVG image is saved as a single printer page. To crop the file, open the PDF file in Mac OS Preview application. Select the "Rectangular Selection" tool and draw a box around the chart. You can adjust the box until it tightly fits the chart. Then click "crop" (or use <kbd>Cmd</kbd>+<kbd>K</kbd>). Make sure to save the newly modified file!
```
It may be worth noting here that this process of converting to PDF using the "Print..." option does require the image to actually fit the size of a printed page (something wider will be cut off). This is potentially an edge case that most people will not have to deal with, but could be worth explaining more about (maybe as a note at the end of the tutorial, in a section like "Tips & Tricks").

Reply:
Great point!